gensim/models/wrappers/ldamallet.py

```diff
@@ -269,7 +280,28 @@ def read_doctopics(self, fname, eps=1e-6, renorm=True):
                            for id_, weight in enumerate(map(float, parts))
                            if abs(weight) > eps]
                 else:
-                    raise RuntimeError("invalid doc topics format at line %i in %s" % (lineno + 1, fname))
+                    if mallet_version == "2.0.7-SNAPSHOT":
+                        # 1   1   0   1.0780612802674239  9   0.005575655428533364    8   0.005575655428533364    7   0.005575655428533364    6   0.005575655428533364    5   0.005575655428533364    30.005575655428533364   2   0.005575655428533364    1   0.005575655428533364    
+                        # 2   2   0   0.9184413079632608  9   0.009062076892971008    8   0.009062076892971008    7   0.009062076892971008    6   0.009062076892971008    5   0.009062076892971008    40.009062076892971008   3   0.009062076892971008    2   0.009062076892971008    1   0.009062076892971008
```

Comment:
Thank you for giving an explanation for this behaviour.
Can you give a shorter example of these 2 formats? Say with num_topics = 3. It will be easier to understand the explanation.